fem/transfer.cpp

```diff
@@ -226,12 +226,63 @@ const Operator &InterpolationGridTransfer::BackwardOperator()
    return *B.Ptr();
 }
 
-
```

Comment:
I think the extra space here was intentional (two newlines between different classes)

Reply:
Ahh, I'll add it back in.  I noticed there are other places in `transfer.cpp` that don't have two newlines between classes.  Should I add an extra newline there as well?  (I haven't looked at other source files)

Reply:
Sure, it would be good if they were there in `transfer.cpp` for consistency.

Reply:
OK, fixed.